spec/features/items/index_spec.rb

```diff
@@ -71,5 +71,121 @@
 
       expect(page).to_not have_content(@muzzle.name)
     end
+
+    it "can list top 5 most popular items and quantity bought" do
+        order_1 = create(:random_order)
+        order_2 = create(:random_order)
+
+        dog_toy = create(:random_item)
+        mug = create(:random_item)
+        boot = create(:random_item)
+        shirt = create(:random_item)
+        hat = create(:random_item)
+        cookie = create(:random_item)
+        pants = create(:random_item)
+
+        ItemOrder.create(item: dog_toy, order: order_1, price: dog_toy.price, quantity: 9)
```

Comment:
Gonna have to mess around with FactoryBot some more to see if there's a better way to write these specs. Good work getting a good test for this though!